bench/performances.js

```diff
@@ -116,12 +116,12 @@ async function serverStart() {
 
   configBuilder.set('store:name', 'local_store');
 
-  configBuilder.set('mapStyle:poiMapUrl', `["http://localhost:${PORT}/fake_pbf/{z}/{x}/{y}.pbf"]`);
-  configBuilder.set('mapStyle:baseMapUrl', `["http://localhost:${PORT}/fake_pbf/{z}/{x}/{y}.pbf"]`);
   configBuilder.set('performance:enabled', true);
```

Comment:
why not move all these overrides to use a single mechanism and remove this confusing usage of  `.set()` ?